### PR TITLE
A descriptive name for my changes

### PR DESCRIPTION
**Pull request by rahul-art**
hind encorp resubmited

### CODE REVIEW

**Comment by lhoestq** [comment]
I have noticed that the master branch of your fork has diverged from the one of the repo. This is probably what causes the mess in the github diff "Files changed".

I would suggest to re-fork the `datasets` repo and recreate a new branch and a new PR.

File: datasets/hind_encorp/hind_encorp.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        # This is the name of the configuration selected in BUILDER_CONFIGS above
+
+        features = datasets.Features(
+            {
+                "id": datasets.Value("string"),
+                "source": datasets.Sequence(datasets.Value("string")),
+                "alignment_type": datasets.Sequence(datasets.Value("string")),
+                "alignment_quality": datasets.Sequence(datasets.Value("string")),
+                "English": datasets.Sequence(datasets.Value("string")),
+                "Hindi": datasets.Sequence(datasets.Value("string"))
```
Comment (lhoestq) [inline datasets/hind_encorp/hind_encorp.py:85]: can you use the `Translation` feature type here ?

You can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)

File: datasets/hind_encorp/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```
Comment (lhoestq) [inline datasets/hind_encorp/README.md:3]: Can you add the yaml tags here ?
You can use the tagger app to do so
More info about how to add the yam tages here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

File: datasets/hind_encorp/hind_encorp.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY-NC-SA 3.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11858/00-097C-0000-0023-625F-0/hindencorp05.plaintext.gz?sequence=3&isAllowed=y"
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class HindEnCorp(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/hind_encorp/hind_encorp.py:61]: HindEnCorp -> HindEncorp

this must be the camel case version of the dataset script name `hind_encorp`

**Review by lhoestq — COMMENTED** [review]
Looking really cool so far ! good job

I left a few comments

**Comment by lhoestq** [comment]
You're pretty close to having all things ready to merge !
Feel free to ping me when you have a new PR

File: datasets/hind_encorp/hind_encorp.py (hunk 1 above)
Comment (rahul-art) [reply datasets/hind_encorp/hind_encorp.py:85]: actually, there is only 1 single file here in which both English and hindi translation are separated by tabs but in forest case there are separate files for different languages

File: datasets/hind_encorp/hind_encorp.py (hunk 3 above)
Comment (rahul-art) [reply datasets/hind_encorp/hind_encorp.py:61]: done

**Comment by lhoestq** [comment]
Closing this one in favor of #1575

**Pull request closed without merging**